test/tests/ha/rackdh_ha_resource_test.py

```diff
@@ -0,0 +1,120 @@
+"""
+Copyright 2017, Dell, Inc.
+
+This test checks the installation of the HA environment
+"""
+
+import fit_common
+
+# LOCAL
+from nose.plugins.attrib import attr
+from nosedep import depends
+
+FAIL_ON_NODE_COLLECTION = True
+active_node_list = list()
+
+
+@attr(regression=False, smoke=False, ha_resource_tests=True)
+class HaResourceTest(fit_common.unittest.TestCase):

```

Comment:
take fit_common. off.  ie. (unittest.TestCase).  Thought we are trying to remove that level of referencing.